src/Moryx.Resources.Management/Resources/ResourceSerialization.cs

```diff
@@ -27,7 +26,7 @@ static ResourceSerialization()
             Serialization = new EntrySerializeSerialization();
         }
 
-        public ResourceSerialization(IContainer container, IRuntimeConfigManager configManager) : base(container, configManager)
+        public ResourceSerialization(IContainer container, IEmptyPropertyProvider configManager) : base(container, configManager)
```

Comment:
I don't think this works nor is it relevant in the future anymore, since resources are serialized outside the module in endpoints now

Reply:
You are right, the internal class didn't have any references anymore 🙈